### PR TITLE
navbar added

### PR DESCRIPTION
**Pull request by qwerty-rgb**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My contribution follows the style guidelines of this project
- [ ] I have performed a self-review of my own contribution
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    </li>
+  ));
+
+  //this methods says how the NavBar should look like in HTML
+  return (
+    <nav className="navbar navbar-expand-lg navbar-light bg-light">
+      <a className="navbar-brand" href="#">
+        {/* here I pass the logo url to the navbar image */}
+        <img src={props.logo} />
+	<b>Women In Technology</b>
```
Comment (qwerty-rgb) [inline src/index.js:24]: Just added a bold heading according to the wireframe

**Review by qwerty-rgb — COMMENTED** [review]
Just added a bold heading

**Comment by shikha-16** [comment]
again, please make the pr to the 'website' branch.

**Comment by qwerty-rgb** [comment]
so i am closing this pr and generating a new pr for more clarity :)

**Pull request closed without merging**